cmd/clusterctl/configuration/ibmcloud/provider-component/user-data/ubuntu/templates/master-user-data.sh

```diff
@@ -92,9 +92,6 @@ cat > /etc/systemd/system/kubelet.service.d/20-kubenet.conf <<EOF
 Environment="KUBELET_DNS_ARGS=--cluster-dns=${CLUSTER_DNS_SERVER} --cluster-domain=${CLUSTER_DNS_DOMAIN}"
 EOF
 
-echo $OPENSTACK_CLOUD_PROVIDER_CONF | base64 -d > /etc/kubernetes/cloud.conf
```

Comment:
Do we need this for ibmcloud?

Reply:
until now, we do not need this `cloud.yaml`. We can access IBM cloud by `userName` and `apiKey`.